src/models/network.js

```diff
@@ -198,11 +198,8 @@ Network.prototype.createIrcFramework = function (client) {
 		"znc.in/self-message", // Legacy echo-message for ZNC
 	]);
 
-	// Request only new messages from ZNC if we have sqlite logging enabled
 	// See http://wiki.znc.in/Playback
-	if (client.messageProvider) {
-		this.irc.requestCap("znc.in/playback");
-	}
+	this.irc.requestCap("znc.in/playback");
```

Comment:
You can merge this into the `this.irc.requestCap` call above, if it accepts an array?